ynput/OpenPype
# Bugfix/add missing houdini settings

## Changelog Description
add missing settings. 
now, it looks like this:
| Ayon | OpenPype |
| -- | -- | 
| ![image](https://github.com/ynput/OpenPype/assets/20871534/3b679aee-4240-4939-bb8d-4f9c84bd3399) | ![image](https://github.com/ynput/OpenPype/assets/20871534/1a5fde23-0a75-484e-9b69-983f7da81101) |
| ![image](https://github.com/ynput/OpenPype/assets/20871534/cd5a339f-18da-4108-890a-1a72e08d9622) | ![image](https://github.com/ynput/OpenPype/assets/20871534/f0188b3f-d6a1-48d0-95aa-711e2473f6a0) |


## Testing notes:
1. Enable and disable mantra ifd from settings
2. change `frames per task` and create a farm publish (point cache bgeo or point cache abc or mantra ifd) and check if frames per cache is the same as n settings.
